pkg/fleet/installer/service/datadog_agent.go

```diff
@@ -113,6 +117,9 @@ func SetupAgent(ctx context.Context) (err error) {
 			return err
 		}
 	}
+	if err = removeOldAgentFiles(); err != nil {
+		return fmt.Errorf("failed to remove old agent files: %v", err)
+	}
```

Comment:
Is it really a good idea to remove deb/rpm managed files manually, without going though the package manager?